.github/ISSUE_TEMPLATE.md

```diff
@@ -1,19 +1,20 @@
 <!--
 Welcome to Zero to JupyterHub!
 
-Before filing an issue, please search through the issues to see
+Before filing an issue, please search through the issues and
+[Discourse forum](https://discourse.jupyter.org/) to see
 if your question has been discussed before. If you
 need more information after searching, feel
 free to message us on the gitter channel. Many
 JupyterHub community members watch the gitter channel
 so you will have the benefit of other users' experience
 as well as the JupyterHub team.
```

Comment:
should we remove the mention of gitter from here or move it to after mentioning discourse? I think it would help with directing more people towards discourse (instead of gitter) which is what we want to achieve.

Reply:
I've removed Gitter, and reworded the rest.